fe/fe-core/src/main/java/org/apache/doris/httpv2/util/streamresponse/StreamResponseInf.java

```diff
@@ -0,0 +1,56 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.httpv2.util.streamresponse;
+
+import java.io.PrintWriter;
+import java.sql.ResultSet;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * StreamResponseInf use response.getWriter() to response client
+ */
+public abstract class StreamResponseInf {
```

Comment:
Why does it end in Inf

Reply:
This class is an interfance of streamResponse, we can extends this abstract class to implement different streamResponse to achieve different outputFormat.
For example, I implenment JSONStreamResonse to output `json` format. If we want to output `csv` format, we can implenment a class called `CSVStreamResponse`.
So, I define this class end in Inf.